README.md

```diff
@@ -72,175 +103,206 @@ To access details of each command:
 ```
 
-- _When creating new Ethereum accounts, they won't have any ETH if not funded and they won't be able to send transactions as they need a minimum of ETH to be able to pay for transaction fees_. In order to add a new faucet to the chain registry, you will need an account that holds some ETH on the Rinkeby testnet. Add the address of the generated account to the `.env` file and make sure you send some ETH to it (you can use MetaMask do that that):
+For example
 
-```.env
-FAUCET_ACCOUNT=<FAUCET_ACCOUNT>
+```bash
+make hashicorp-vault COMMAND="token lookup"
+```
+
+## Connect a blockchain network
+
+### Register blockchain network
+
+Use Orchestrate API to register a blockchain network, you should provide
+
+- `name` for the network
+- `urls` of at least one JSON-RPC endpoint of an Ethereum node in the network
+
+> **Note:** _We will be connecting to Rinkeby through [Infura](https://infura.io/) but you can connect to any Ethereum network using same procedure_
+
+```bash
+curl -X POST --data '{"name": "rinkeby", "urls":["https://rinkeby.infura.io/v3/<INFURA_PROJECT_ID>"]}' localhost:8081/chains
 ```
 
-To add a new chain to the chain registry and start listening to transactions coming from that chain, run the following command:
+<img src="static/post-chain.png" width="900px" alt="Create Chain"/>
+
+> **Note:** _Save chain unique identifier `uuid` for later usage_
+
+### Send JSON-RPC request through blockchain proxy
+
+Verify that chain JSON-RPC is properly proxied by Orchestrate
 
 ```bash
-npm run register-chain
+curl -X POST --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["latest", false],"id":1}' localhost:8081/<CHAIN_UUID>
 ```
 
-by doing that, we connect Orchestrate to the Rinkeby tesnet though Infura and register a new faucet.
+<img src="static/json-rpc-proxy.png" width="900px" alt="JSON-RPC Response"/>
+
+## Configure a Faucet
 
-> Have a look at the script `src/register-chain/register.ts` for reference
+> **Note:** _On non zero gas price networks (e.g. public networks such as mainnet), an Ethereum account must be pre-funded with ETH to be able to send transaction. Orchestrate Faucet allows to automatically prefund accounts managed in Orchestrate_
 
-### Smart Contract management
+Use Orchestrate API to configure a Faucet, you should provide
 
-Use Truffle (already installed) to compile your smart contracts into the `build` folder:
+- `name` for the Faucet
+- `creditorAccount` that will be used to credit other accounts (account should be managed by Orchestrate)
+- `chainRule` chain unique unique identifier
+- Faucet configuration (c.f. Orchestrate documentation for more details)
+
+> **Note:** _Set `creditorAccount` and `chainRule` with values you saved in prior sections of this Quick Start_
+
+> **Warning:** _Faucet account must be credited with some ETH. To do so, you can use a Faucet (e.g. https://faucet.rinkeby.io/ if using Rinkeby network) or send a ETH transfer transaction using Metamask_
```

Comment:
Should be indicated in the requirements that you need a Twitter or FB account and it should not be in a warning but a clear mandatory step indicating the process and how many ethers to request